internal/service/project/resource_project.go

```diff
@@ -217,6 +219,13 @@ func (r *projectRS) Schema(ctx context.Context, req resource.SchemaRequest, resp
 					boolplanmodifier.UseStateForUnknown(),
 				},
 			},
+			"is_slow_operation_thresholding_enabled": schema.BoolAttribute{
+				Computed: true,
```

Comment:
Needs to be computed since the backend provides a "default" value when it is not set